docs/reference/tags/easol_badge_tag/index.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Easol Badge Tag
+parent: Tags
+has_children: false
+---
+
+The Easol Badge Tag returns a HTML <a> Tag linking to easol.com
```

Comment:
Maybe wrap the <a> element, otherwise it renders weirdly on the browser.

```suggestion
The Easol Badge Tag returns a HTML `<a>` Tag linking to easol.com
```